source/jormungandr/jormungandr/default_settings.py

```diff
@@ -109,3 +109,5 @@
     
 GRAPHICAL_ISOCHRONE = False
 HEAT_MAP = False
+
+PATCH_WITH_GEVENT_SOCKET = False
```

Comment:
no new line, and the name is a bit confusing, maybe more something with parallel_street_network_calls?